### PR TITLE
feat(sig-services): add security implications requirement

### LINKED ISSUE #108: [SIG Services][Guideline] Explain the security implications
Explain the security implications of using the service so the user can understand and assess the risk 

Why do we require these privileges to impersonate the user/act on their behalf?

Recording: https://drive.google.com/file/d/1eMIMtNLV48AuqFdH2RSckyWBhKSme9lf/view?usp=sharing

### PR DESCRIPTION
**Pull request by SamoKopecky**
Closes #108

### CODE REVIEW

File: sig-services/service-requirements/security-implications.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: Define and explain the security implications
+persona: service provider/owner
+phase: service-design, service-operational
+---
+
+The service should have defined security implications that are applied when a user subscribes to the service. The document that is describing these security implications should also define why are they required for the full functionality of the service.
```
Comment (tumido) [inline sig-services/service-requirements/security-implications.md:7]: I would rephrase this, security implication isn't something that "is applied" I think.. it's rother something that "happens"...

```suggestion
The service should disclose all security implications that can have an impact on a user which subscribes to the service. These implications are listed in a document which describes in detail what is the risk and why it is necessary for the full functionality of the service. In case there's an option to opt-out of a particular risk factor the document states what effect it has on service functionality.
```

File: sig-services/service-requirements/security-implications.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Requirements
+
+- Define the security implications and explain why they are required in a form (a separate document or a prompt before subscription to the service) easily accessible by the user
```
Comment (tumido) [inline sig-services/service-requirements/security-implications.md:11]:
```suggestion
- Define and disclose all the security implications and explain why they are required in a form (a separate document or a prompt before subscription to the service) easily accessible by the user
```

File: sig-services/service-requirements/security-implications.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+## Reasoning
+
+Defining and explaining the security implications can help the user with assessing the security risk associated with using the service.
```
Comment (tumido) [inline sig-services/service-requirements/security-implications.md:15]:
```suggestion
Defining and explaining the security implications can help the user with assessing the security risk associated with using the service. It boosts service transparency and rationalize user expecations.
```

**Review by tumido — CHANGES_REQUESTED** [review]

File: sig-services/service-requirements/security-implications.md (hunk 1 above)
Comment (SamoKopecky) [reply sig-services/service-requirements/security-implications.md:7]: Agree, the opt-out option is also a good idea.

**Comment by SamoKopecky** [comment]
Thanks for the feedback, changes are applied.

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Review by Shreyanand — APPROVED** [review]
/approve
